parsec/core/invite/organization.py

```diff
@@ -66,5 +67,9 @@ async def bootstrap_organization(
         redacted_user_certificate=redacted_user_certificate,
         redacted_device_certificate=redacted_device_certificate,
     )
+    if rep["status"] in ("not_found", "already_bootstrapped"):
+        raise InviteNotAvailableError()
```

Comment:
Can we have another exception ? In the GUI it makes sense to be able to differentiate these cases and display an error message accordingly.